### ISSUE TITLE
Problem with nlmixr in windows

### ISSUE THREAD

**Issue by OmarAshkar** [pharmpy/pharmr#143]
Hi,

I have tested to run with nlmixr on my windows PC but I get the following error: 

```
2023-08-22 20:09:20,964 - distributed.worker - WARNING - Compute Failed
Key:       run0-results
Function:  execute_task
args:      ((<function post_process_results_many at 0x0000023007181480>, <pharmpy.workflows.tool_database.local_directory.LocalDirectoryToolDatabase object at 0x0000023008BAB070>, (<function retrieve_from_database_or_execute_model_with_tool.<locals>.task at 0x0000023008BC5EA0>, <pharmpy.workflows.tool_database.local_directory.LocalDirectoryToolDatabase object at 0x0000023008BAB070>, <Pharmpy model object pheno>)))
kwargs:    {}
Exception: "OSError(22, 'The filename, directory name, or volume label syntax is incorrect')"
```

The code was 
```
model <- load_example_model('pheno')
model <- convert_model(model, "nlmixr")
results <- fit(model, tool = "nlmixr")
```

Thanks alot! 
Omar

**Comment by rikardn** [comment]
Thanks for reporting this. @JohanWBorg: how do you configure Pharmpy to find R and nlmixr when fitting?

**Comment by JohanWBorg** [comment]
Hi!
Thanks so much for reporting this! It seems you caught a small bug which causes the issue and should be fixed as of [PR #1984](https://github.com/pharmpy/pharmpy/pull/1984)

However, if not already set up. You might also need to configure nlmixr/R in pharmpy which is done through Pharmpy's configuration file (pharmpy.conf). The location of said file can be found using  `pharmpy.modeling.get_config_path()`. If one has not been created, I believe the required path should be given in a warning message here. 
It should look something like `C:\Users\USERNAME\AppData\Local\Pharmpy\Pharmpy\pharmpy.conf`

To set up nlmixr/r support, the path to the installation of R needs to be given in this file as seen below.
```
[pharmpy.plugins.nlmixr]
rpath=path_to_r
```
It probably looks something like `C:\Program Files\R\R-4.3.1`

**Comment by rikardn** [comment]
The PR has been merged so the fix will arrive in the next release (which will be 0.100). Please open a new issue if there are problems with the new version. Thanks @OmarAshkar for reporting and @JohanWBorg for fixing!
